tools/ci/src/args.rs

```diff
@@ -0,0 +1,5 @@
+#[derive(Copy, Clone, PartialEq, Eq)]
+pub struct Args {
```

Comment:
(Nit) Could you add some docs to this struct, since the rest of the programs deals with it a lot? A few things I would cover:

1. What is its purpose? (To act as a `Prepare`-friendly version of `CI`)
2. Highlight how this needs to be kept up-to-date with `CI`.